routes/api_camps_routes.js

```diff
@@ -1217,15 +1217,15 @@ module.exports = (app, passport) => {
                         // stat.last_24h_entrance = result[0][0]['last_24h_entrance'];
                     }).then(() => {
                         knex.raw(query1).then((result) => {
-                            stat.total_tickets = result[0][0]['total_tickets'];
-                            stat.inside_event = result[0][0]['inside_event'];
-                            stat.ticketing = result[0][0]['ticketing'];
-                            stat.last_24h_first_entrance = result[0][0]['last_24h_first_entrance'];
-                            stat.last_1h_first_entrance = result[0][0]['last_1h_first_entrance'];
-                            stat.last_24h_entrance = result[0][0]['last_24h_entrance'];
-                            stat.last_24h_exit = result[0][0]['last_24h_exit'];
-                            stat.last_1h_entrance = result[0][0]['last_1h_entrance'];
-                            stat.last_1h_exit = result[0][0]['last_1h_exit'];
+                            stat.total_tickets = result[0]['total_tickets'];
+                            stat.inside_event = result[0]['inside_event'];
+                            stat.ticketing = result[0]['ticketing'];
+                            stat.last_24h_first_entrance = result[0]['last_24h_first_entrance'];
+                            stat.last_1h_first_entrance = result[0]['last_1h_first_entrance'];
+                            stat.last_24h_entrance = result[0]['last_24h_entrance'];
+                            stat.last_24h_exit = result[0]['last_24h_exit'];
+                            stat.last_1h_entrance = result[0]['last_1h_entrance'];
+                            stat.last_1h_exit = result[0]['last_1h_exit'];
                             res.status(200).json({ groups: groups, stats: stat });
```

Comment:
should prevent the dump of:
{"isFulfilled":false,"isRejected":true,"rejectionReason":{**"message":"Cannot read property 'total_tickets' of undefined"**,"stack":"TypeError: Cannot read property 'total_tickets' of undefined\n    at knex.raw.then (/home/yaniv/workspace/Spark/routes/api_camps_routes.js:1220:62)\n    at tryCatcher(/home/yaniv/workspace/Spark/node_modules/bluebird/js/release/util.js:16:23)\n    at Promise._settlePromiseFromHandler (/home/yaniv/workspace/Spark/node_modules/bluebird/js/release/promise.js:512:31)\n    at Promise._settlePromise (/home/yaniv/workspace/Spark/node_modules/bluebird/js/release/promise.js:569:18)\n    at Promise._settlePromise0 (/home/yaniv/workspace/Spark/node_modules/bluebird/js/release/promise.js:614:10)\n    at Promise._settlePromises (/home/yaniv/workspace/Spark/node_modules/bluebird/js/release/promise.js:693:18)\n    at Async._drainQueue (/home/yaniv/workspace/Spark/node_modules/bluebird/js/release/async.js:133:16)\n    at Async._drainQueues (/home/yaniv/workspace/Spark/node_modules/bluebird/js/release/async.js:143:10)\n    at Immediate.Async.drainQueues (/home/yaniv/workspace/Spark/node_modules/bluebird/js/release/async.js:17:14)\n    at runCallback (timers.js:672:20)\n    at tryOnImmediate (timers.js:645:5)\n    at processImmediate [as _immediateCallback] (timers.js:617:5)"}}